pytorch_toolkit/README.md

```diff
@@ -21,7 +21,8 @@ After installation, you are ready to train your own models, evaluate and use
 them for prediction.
 
 * [Action Recognition](action_recognition)
-* [ASL Recognition](asl_recognition)
+* [Action Recognition 2](action_recognition_2)
```

Comment:
Is it worth adding '2' if we use it as `domain` containing different `problems`?

Reply:
The original `ASL Recognition` is not under model templates and it doesn't use any mmaction... so the environment is different.